libraries/WiFiClientSecure/src/WiFiClientSecure.cpp

```diff
@@ -393,6 +419,8 @@ int WiFiClientSecure::setTimeout(uint32_t seconds)
 }
 
 int WiFiClientSecure::fd() const
+
+int WiFiClientSecure::setSocketOption(int option, char* value, size_t len)
```

Comment:
something must have happened here? This line belongs elsewhere IMHO

Reply:
Should be good now.